CODEOWNERS

```diff
@@ -23,6 +23,7 @@
 /addons/binding/org.openhab.binding.feed/ @svilenvul
 /addons/binding/org.openhab.binding.feed.test/ @svilenvul
 /addons/binding/org.openhab.binding.ftpupload/ @paulianttila
+/addons/binding/org.openhab.binding.hdpowerview/ @beowulfe
```

Comment:
`hdpowerview` should not be added. The next line also on `groheondus` should be deleted.